dynatrace_topology/tests/samples/application_response_topology.json

```diff
@@ -0,0 +1,146 @@
+{
+  "components": [
+    {
+      "data": {
+        "entityId": "APPLICATION-EA7C4B59F27D43EB",
+        "displayName": "RUM Default Application",
+        "customizedName": "RUM Default Application",
+        "discoveredName": "RUM Default Application",
+        "firstSeenTimestamp": 1422282024216,
+        "lastSeenTimestamp": 1538579528065,
+        "applicationType": "DEFAULT",
+        "ruleAppliedPattern": "http",
+        "managementZones": [
+          {
+            "id": "-6239538939987181652",
+            "name": "allTypes"
+          },
+          {
+            "id": "-2519468841583898843",
+            "name": "app name exists"
+          },
+          {
+            "id": "4485554873951847460",
+            "name": "Applications except easyTravel"
+          }
+        ],
+        "domain": "",
+        "environment": "production",
+        "identifiers": [
+          "urn:application:/APPLICATION-EA7C4B59F27D43EB"
+        ],
+        "instance": "https://ton48129.live.dynatrace.com",
+        "tags": [
+          "integration-type:dynatrace",
+          "integration-url:https://ton48129.live.dynatrace.com"
+        ],
+        "labels": [
+          "dynatrace-managementZones:allTypes",
+          "dynatrace-managementZones:app name exists",
+          "dynatrace-managementZones:Applications except easyTravel",
```

Comment:
Does stackstate support labels with spaces? Perhaps we need to sanitize label data for special characters?

Reply:
it does as I saw this in AXA environment with spaces.